src/pages/user-projects/user-projects.less

```diff
@@ -0,0 +1,7 @@
+#userProjects {
+  display: block;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  padding: 20px 0;
+}
```

Comment:
Yeah, I thought about doing that but didn't want to go too far down the route of optimizing the CSS, as I wanted to focus on the main feature, especially since the main less file is broken down by pages.